src/cow-react/modules/wallet/web3-react/connection/ledger.tsx

```diff
@@ -26,7 +26,7 @@ const [ledger, ledgerHooks] = initializeConnector<AsyncConnector>(
               return new m.Ledger({
                 actions,
                 options: {
-                  rpc: RPC_URLS,
+                  rpc: PROVIDERS_RPC_URLS_FIRST_ONLY,
```

Comment:
nit: we could add the comment in the same line as the one of coinbase explaining that ledger allows just one RPC per network